libjulia/backends/evm/EVMCodeTransform.h

```diff
@@ -78,22 +86,20 @@ class CodeTransform: public boost::static_visitor<>
 	CodeTransform(
 		julia::AbstractAssembly& _assembly,
 		solidity::assembly::AsmAnalysisInfo& _analysisInfo,
+		Block const& _block,
+		solidity::AssemblyStack::Language _language,
+		bool _allowStackOpt,
```

Comment:
I'm not sure about this variable name. Why not `_reuseStackSlots`? Also cannot this just depend on the `_language`?

Reply:
It can be controlled by language directly, I think @chriseth also wants a boolean to control it.